manual/src/refman/extensions/alerts.etex

```diff
@@ -17,30 +17,64 @@ component is referenced).
 
 The "ocaml.alert" or "alert" attribute serves two purposes: (i) to
 mark component with an alert to be triggered when the component is
-referenced, and (ii) to control which alert names are enabled.  In the
-first form, the attribute takes an identifier possibly
+referenced, and (ii) to control which alert names are enabled.
+
+\subsection{ss:marking}{Marking component with an alert}
```

Comment:
I would propose `ss:alert_marking` because label names are shared on the whole manual.

Also  maybe `Marking a component`?

Reply:
Ah, I didn't expect the names to be global. I'm made the changes, thanks.